packages/flutter_tools/test/commands.shard/hermetic/daemon_test.dart

```diff
@@ -443,7 +469,7 @@ void main() {
     });
 
     testWithoutContext('does not run any operations concurrently', () async {
-      // Crete a function thats slow, but throws if another instance of the
+      // Crete a function that's slow, but throws if another instance of the
```

Comment:
nit: Create* 😅

Reply:
uhh, if you wanna send a PR for this I'd be happy to review :)